jaeger-jaxrs2/src/main/java/com/uber/jaeger/filters/jaxrs2/TracingUtils.java

```diff
@@ -18,22 +18,32 @@
 import io.opentracing.Tracer;
 import java.util.concurrent.ExecutorService;
 
+/**
+ * TracingUtils is going to be deprecated. To use its method please provide a tracer via
+ * {@link com.uber.jaeger.filters.jaxrs2.TracingUtils#setTracer(io.opentracing.Tracer)}.
+ */
+@Deprecated
 public class TracingUtils {
+
+  public static synchronized void setTracer(io.opentracing.Tracer tracer) {
+    com.uber.jaeger.context.TracingUtils.setTracer(tracer);
+  }
+
   @Deprecated
-  public static TraceContext getTraceContext() {
+  public static synchronized TraceContext getTraceContext() {
```

Comment:
These methods don't need to by synchronized because the underlying `context.TracingUtils` class is thread safe.